dustmaps/sfd.py

```diff
@@ -93,43 +93,84 @@ def query(self, coords, order=1):
 
 class SFDQuery(SFDBase):
     """
-    Queries the Schlegel, Finkbeiner & Davis (1998) dust reddening map.
+    Queries maps stored in the same format as Schlegel, Finkbeiner & Davis (1998) dust reddening map, providing results from the Schlegel, Finkbeiner & Davis (1998) dust map by default.
     """
 
     map_name = 'sfd'
     map_name_long = "SFD'98"
 
-    def __init__(self, map_dir=None):
+    def __init__(
+        self, 
+        map_dir=None,
+        whichparent='SFD',
+        which='dust'
+    ):
         """
         Args:
             map_dir (Optional[str]): The directory containing the SFD map.
                 Defaults to `None`, which means that `dustmaps` will look in its
                 default data directory.
+            whichparent (Optional[:obj:`str`]): The parent name of the SFD style data product to download.
+                Should be either ``SFD`` (the default), ``Synch``, ``FINK``, or ``Haslam``.
+            which (Optional[:obj:`str`]): The name of the SFD style data product to download.
+                Should be either ``dust`` (the default), ``i100``, ``i60``, ``mask``, ``temp``, or ``xmap`` with whichparent ``SFD``. Should be ``Beta`` for whichparent ``Synch``. Should be ``Rmap`` for whichparent ``FINK``. Should be ``clean`` for whichparent ``Haslam``.
         """
         
         if map_dir is None:
             map_dir = os.path.join(data_dir(), 'sfd')
 
-        base_fname = os.path.join(map_dir, 'SFD_dust_4096')
+        if (whichparent == 'SFD') and (which == 'dust' or which == 'i100' or which == 'i60' or which == 'mask'):
```

Comment:
The second condition could be simplified to `(which in ('dust', 'i100', 'i60', 'mask'))`.

Reply:
I find that harder to read, but it is more pythonic. I will make that change.

Reply:
There was one other location where I have now made the same replacement